packages/@lwc/perf-benchmarks-components/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "2.30.1",
     "private": true,
     "scripts": {
-        "build": "rm -fr dist && rollup -c"
+        "build": "rm -fr dist && rollup -c ./rollup.config.mjs"
```

Comment:
Rollup now requires config files to be in actual ESM format. I could add `"type": "module"` but then I'd have to upgrade every script. `.mjs` is easier.

Reply:
Similar question, the three `@lwc/engine-xxx` packages also use `rollup.config.js` files, how does it work there?